draft-ietf-quic-recovery.md

```diff
@@ -722,14 +695,13 @@ Pseudocode for OnAckReceived and UpdateRtt follow:
       // Find the smallest newly acknowledged packet
       smallest_newly_acked =
         FindSmallestNewlyAcked(newly_acked_packets)
-      // If any packets sent prior to RTO were acked, then the
-      // RTO was spurious. Otherwise, inform congestion control.
-      if (rto_count > 0 &&
-            smallest_newly_acked > largest_sent_before_rto):
-        OnRetransmissionTimeoutVerified(smallest_newly_acked)
+      // If any packets sent prior to PTO were acked, then the
+      // PTO was spurious. Otherwise, inform congestion control.
+      if (pto_count > 0 &&
+            smallest_newly_acked > largest_sent_before_pto):
```

Comment:
You could define a kNumCongestivePtoPackets here and add it here and then define it above to be the number of PTOs you will treat as standard packet loss, and not extreme congestion.

Reply:
What is "extreme congestion" ?

Reply:
@martinthomson : @ianswett is trying to capture the difference in congestion response that the current loss recovery spec has for TLP vs. RTO. The idea is to say that for a couple of PTOs, there will be a smaller reduction in cwnd, and after that, the cwnd will be collapsed. I can write that up.